src/services/casbin/casbin.conf

```diff
@@ -0,0 +1,14 @@
+[request_definition]
```

Comment:
what is the difference between what settings will be stored here versus in mongodb?

Reply:
MongoDB stores the individual policies - this file defines the logic of how the policies are interpreted. Its like a template for all authorization logic.

Casbin is very flexible and supports up to 6 fields per policy that can be used to determine authorization. Also you can define multiple rules of how policies apply so you can represent things like inheritance etc.

Reply:
is it an out of the box model or a modified one? if the first, maybe we could use the filename by default on https://casbin.org/docs/en/supported-models

Reply:
sounds to be this model: https://github.com/casbin/casbin/blob/master/examples/rbac_with_domains_model.conf ?

Reply:
Its not exactly that model.
The one you are referencing requires 4 arguments:
```
[request_definition]
r = sub, dom, obj, act
```

Meaning to test authorization you need to also pass in the specific domain.
In the modified version that I am proposing casbin would figure out by itself wether a permission exists via any domain. That way we only need to pass in 3 arguments which makes things a lot simpler.